WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewController.swift

```diff
@@ -38,7 +38,7 @@ final class IssueRefundViewController: UIViewController {
 // MARK: Actions
 private extension IssueRefundViewController {
     @IBAction func nextButtonWasPressed(_ sender: Any) {
-        print("Next button pressed")
+        navigationController?.pushViewController(RefundConfirmationViewController(), animated: true)
```

Comment:
```suggestion
        show(RefundConfirmationViewController(), animated true)
```
We can use `show` to decouple ourselves a bit from the optional navigation controller.

Reply:
Thanks! TIL. 

Fixed in bd759a67aecf50d3d7e05da86c9375353db1c65f.